streampipes-rest/src/main/java/org/apache/streampipes/rest/impl/datalake/DataLakeResource.java

```diff
@@ -416,6 +417,14 @@ private boolean checkProvidedQueryParams(Map<String, String> providedParams) {
   public ResponseEntity<?> setDataLakeRetention(
       @PathVariable String elementId,
       @RequestBody RetentionTimeConfig retention){
+        try {
+          
+          retention.getRetentionExportConfig().getExportProviderSettings().setSecretKey(
+          SecretEncryptionManager.encrypt(retention.getRetentionExportConfig().getExportProviderSettings().getSecretKey()));
+          
+        } catch (Exception e) {
+          e.printStackTrace();
```

Comment:
Use `LOG.error` do log errors instead of printing the stack trace
Do we want to return a bad request/internal server error in this case?